modules/ocf_irc/manifests/ircd.pp

```diff
@@ -55,5 +55,18 @@
       content   => $irc_creds['cert_reload_password'],
       mode      => '0640',
       show_diff => false;
+
+    '/etc/systemd/system/inspircd.service.d/':
+      ensure => directory,
+      owner  => root,
+      group  => root,
+      mode   => '0755';
+
+    '/etc/systemd/system/inspircd.service.d/override.conf':
```

Comment:
Consider using `ocf::systemd::override`?

Reply:
Good point forgot about that. Thanks.